weed/server/filer_server_handlers_read.go

```diff
@@ -100,6 +100,16 @@ func (fs *FilerServer) GetOrHeadHandler(w http.ResponseWriter, r *http.Request,
 		w.Header().Set(k, string(v))
 	}
 
+	//Add some custom headers that need to be exposed
```

Comment:
"need to be exposed" is not clear.

How about "Seaweed custom header are not visible to Vue or javascript?"